tools/generate-pdfs.sh

```diff
@@ -16,7 +16,28 @@ set -ex
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+function generate_rst_specs() {
+    ./tools/gen-intrinsics-specs.py --intrinsic-defs $1 \
+			    --classification $2 \
+			    --template $3 --outfile $4 \
+			    --workflow $5
+}
+
 function generate_pdfs_from_md() {
+	
+	# Generate tmp file for pdfs
+	generate_rst_specs ./tools/intrinsic_db/mve.csv \
+			   ./tools/intrinsic_db/mve_classification.csv \
+			   ./mve_intrinsics/mve.template.md \
+			   ./tmp/mve.for-pdf.md \
+			   pdf
+
+	generate_rst_specs ./tools/intrinsic_db/advsimd.csv \
+	 		   ./tools/intrinsic_db/advsimd_classification.csv \
+	 		   ./neon_intrinsics/advsimd.template.md \
+	 		   ./tmp/advsimd.for-pdf.md \
+	 		   pdf
+
```

Comment:
I guess I overlooked the fact that the generate_rst_specs is needed only for MVE and NEON... sorry. Also, I just realized that we had the `WARNING` messages for missing files... 

Apologies, this was a mistake on my side. Please restore the code as it was before (all md files, even the ones for the PDFs, generated with `tools/generate-intrinsics-specs.sh `)